apps/electron-app/src/renderer/src/components/ui/context-menu-detector.tsx

```
@@ -0,0 +1,141 @@
+    if (
+      target.tagName === "INPUT" ||
+      target.tagName === "TEXTAREA" ||
+      target.contentEditable === "true" ||
+      target.closest('input, textarea, [contenteditable="true"]')
+    ) {
+      info.isInput = true;
+    }
+
+    // Check if it's a link
+    const linkElement = target.closest("a");
+    if (linkElement && linkElement.href) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)